nrf_802154/serialization/include/host/nrf_802154.h

```diff
@@ -106,7 +106,7 @@ bool nrf_802154_ack_data_set(const uint8_t       * p_addr,
  *   - For Standard-compliant, @ref NRF_802154_SRC_ADDR_MATCH_ALWAYS_1
  * For more information, see @ref nrf_802154_src_addr_match_t.
  *
- * The method can be set during initialization phase by calling @ref nrf_802154_src_matching_method.
+ * The method can be set during initialization phase by calling @ref nrf_802154_ack_data_src_addr_matching_method_set.
```

Comment:
```suggestion
 * The method can be set during initialization phase by calling @ref nrf_802154_src_addr_matching_method_set.
```
Applies to all subsequent occurrences of `nrf_802154_ack_data_src_addr_matching_method_set` in this file as well.